### DISCUSSION TITLE
Add to the CSP FAQ on how to do teleport

### DISCUSSION THREAD

**Discussion by TrustNoOneElse** [FirstGearGames/FishNet#216] [Documentation]
Its not the most common questions, but for sure be helpful to know for them, that they need to clear the cache, to be able to teleport predicted objects with CSP and might need to deactivate there movement for a short duration.

**Comment by FirstGearGames** [comment]
Accepted.

**Comment by FirstGearGames** [comment]
Bump.

**Comment by FirstGearGames** [comment]
Added https://fish-networking.gitbook.io/docs/manual/guides/client-side-prediction/version-1/more/teleporting

**Reply by TrustNoOneElse** [reply]
Wuhu! :D

**Reply by lordubbe** [reply]
> Added https://fish-networking.gitbook.io/docs/manual/guides/client-side-prediction/version-1/more/teleporting

Link is broken - how the heck do you teleport?
